internal/schedule_client.go

```diff
@@ -597,8 +599,7 @@ type (
 		// methods like ScheduleHandle.Describe() will return an error.
 		GetHandle(ctx context.Context, scheduleID string) ScheduleHandle
 	}
-
 )
 
-func (ScheduleWorkflowAction) isScheduleAction() {
+func (*ScheduleWorkflowAction) isScheduleAction() {
```

Comment:
Note that this is a backwards incompatible change. I think it's ok because we're "experimental", just worth noting.